### DISCUSSION TITLE
Plotters don't show up in the GUI?

### DISCUSSION THREAD

**Discussion by zehm** [swar/Swar-Chia-Plot-Manager#137] [Q&A]
I just got my system set up, it appears to be working. I started the manager, then started the viewer, and it shows that several plots are being plotted.

```
=================================================================================
1     ?       None   51331   2021-05-06 15:12:00   06:11:15       3       02:43 / 01:14   90.81%     0 GiB    
2     2tb1    32     80776   2021-05-06 21:02:46   00:20:29       1                       6.67%      91 GiB   
3     2tb2    32     80777   2021-05-06 21:02:46   00:20:29       1                       6.59%      90 GiB   
4     512gb   32     80778   2021-05-06 21:02:46   00:20:29       1                       5.17%      78 GiB   
=================================================================================

```

The first plot (with the ? name) is one I started from the GUI. The other three are started with Swar-Plot-Manager. The thing that feels odd is that the GUI doesn't show any of the plotters started by Swar. Is that normal? I would have exptected the GUI to show all of the plotters running so just wanted to confirm I didn't screw something up?

Thanks for a great tool!

**Comment by ckocank** [comment]
that's normal, I think
